pkg/deviceclaimingserver/enddevices/testdata/config.yml

```diff
@@ -1,5 +1,10 @@
 join-servers:
-  - file: localhost.yml
+  - file: ttjsv1-localhost.yml
     join-euis:
       - 800000000000000C/64
     type: ttjs
```

Comment:
This can be removed along with `pkg/deviceclaimingserver/enddevices/testdata/ttjsv1-localhost.yml`right?